src/govuk/components/accordion/accordion.test.js

```diff
@@ -118,13 +118,24 @@ describe('/components/accordion', () => {
         expect(expandedState).toEqual(expandedStateAfterRefresh)
       })
 
-      describe('"+/-" icon', () => {
-        it('should display "+/-" in the section headings', async () => {
+      describe('"↓/↑" icon', () => {
+        it('should display "↓/↑" in the section headings', async () => {
           await page.goto(baseUrl + '/components/accordion/preview', { waitUntil: 'load' })
 
           const numberOfExampleSections = 2
 
-          const numberOfIcons = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-accordion__icon').length)
+          const numberOfIcons = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-accordion-nav__chevron').length)
+          expect(numberOfIcons).toEqual(numberOfExampleSections)
+        })
+      })
+
+      describe('hidden text', () => {
+        it('should contain hidden text " this section" for assistive technology', async () => {
+          await page.goto(baseUrl + '/components/accordion/preview', { waitUntil: 'load' })
+
+          const numberOfExampleSections = 2
+
+          const numberOfIcons = await page.evaluate(() => document.body.querySelectorAll('.govuk-accordion .govuk-accordion__section .govuk-visually-hidden').length)
```

Comment:
It would be good to check that the visually hidden div contains the " this section" text. Similar to what we do with the "Show all sections" button.